src/model/channels.js

```diff
@@ -28,21 +43,18 @@ const RouteDef = {
   primary: Boolean,
   username: String,
   password: String,
-  type: {
-    type: String,
-    default: 'http',
-    enum: ['http', 'tcp', 'mllp']
-  },
-  cert: Schema.Types.ObjectId,
-  status: {
-    type: String,
-    default: 'enabled',
-    enum: ['enabled', 'disabled']
-  },
   forwardAuthHeader: {
     type: Boolean,
     default: false
-  }
+  },
+  waitPrimaryResponse: Boolean,
+  statusCodesCheck: String,
+  kafkaClientId: String,
+  // Kafka route definition
+  kafkaBrokers: {
```

Comment:
I think the broker should be a global configuration coming from the config file. I don't think these will need to be different per channel.